okio/src/main/kotlin/okio/Sink.kt

```diff
@@ -15,4 +15,8 @@
  */
 package okio
 
-expect interface Sink
+expect interface Sink {
+  fun write(source: Buffer, byteCount: Long)
+
```

Comment:
Missing close and flush

Reply:
Those come from Java interfaces so keeping them in JVM for now. Can also introduce our own interfaces and typealias to Java ones, would that make sense?

Reply:
On the JVM we use the interfaces to carry forward the declarations, but those methods are still intrinsic to the type and need defined here. What is a sink or a source that you can't call close on?